### ISSUE TITLE
Deleting a user in an environment with `allowAdminChanges` set to false results in an error

### ISSUE THREAD

**Issue by jalendport** [craftcms/cms#6270]
### Description
I'm trying to delete a user in an environment where `allowAdminChanges` is set to false, and I'm seeing the _"Unauthorized: Administrative changes are disallowed in this environment."_ screen while doing so. It shows up after I'm asked whether to assign their content or remove it, and the user I'm trying to delete is an admin (in case that's helpful to debug).

### Steps to reproduce

1. Set `allowAdminChanges` to false
2. Try to delete an admin user

### Additional info

- Craft version: Craft Pro 3.4.21
- PHP version: 7.3.19
- Database driver & version: MySQL 5.7.30
- Plugins & versions:

| Plugin | Version |
|---|---|
| AsyncQueue | 2.1.1 |
| Commerce Widgets | 2.0.21 |
| Control Panel Body Classes | 2.2.0 |
| Control Panel CSS | 2.2.1 |
| Craft Commerce | 3.1.5 |
| Element API | 2.6.0 |
| Environment Label | 3.2.0 |
| Expanded Singles | 1.1.2 |
| Field Manager | 2.1.6 |
| Hide Admin | 1.1.0 |
| Lockout | 1.0.1 |
| Mailgun | 1.4.3 |
| Order Notes | 2.0.2 |
| Recurring Orders | 0.0.28 |
| Redactor | 2.6.1 |
| Reverse Relations | 1.2.1 |
| Stripe for Craft Commerce | 2.3.0 |
| Super Table	| 2.4.9 |
| Twigpack | 1.2.2 |
| Twig Perversion	| 2.1.0 |
| Walk | 3.0.0 |

**Comment by brandonkelly** [comment]
I’m not able to reproduce that. Are you seeing the error in your logs? If so please post the stack trace that follows it.

**Comment by jalendport** [comment]
@brandonkelly 

```
2020-06-30 10:59:33 [-][1][b5a9fac2a46ec054a775dc948a7c8e8d][error][yii\web\HttpException:403] yii\web\ForbiddenHttpException: Administrative changes are disallowed in this environment. in /srv/users/serverpilot/apps/main-production/vendor/craftcms/cms/src/web/Controller.php:290
Stack trace:
#0 /srv/users/serverpilot/apps/main-production/vendor/craftcms/cms/src/controllers/UsersController.php(1475): craft\web\Controller->requireAdmin()
#1 [internal function]: craft\controllers\UsersController->actionDeleteUser()
#2 /srv/users/serverpilot/apps/main-production/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#3 /srv/users/serverpilot/apps/main-production/vendor/yiisoft/yii2/base/Controller.php(157): yii\base\InlineAction->runWithParams(Array)
#4 /srv/users/serverpilot/apps/main-production/vendor/craftcms/cms/src/web/Controller.php(178): yii\base\Controller->runAction('delete-user', Array)
#5 /srv/users/serverpilot/apps/main-production/vendor/yiisoft/yii2/base/Module.php(528): craft\web\Controller->runAction('delete-user', Array)
#6 /srv/users/serverpilot/apps/main-production/vendor/craftcms/cms/src/web/Application.php(291): yii\base\Module->runAction('users/delete-us...', Array)
#7 /srv/users/serverpilot/apps/main-production/vendor/craftcms/cms/src/web/Application.php(559): craft\web\Application->runAction('users/delete-us...', Array)
#8 /srv/users/serverpilot/apps/main-production/vendor/craftcms/cms/src/web/Application.php(270): craft\web\Application->_processActionRequest(Object(craft\web\Request))
#9 /srv/users/serverpilot/apps/main-production/vendor/yiisoft/yii2/base/Application.php(386): craft\web\Application->handleRequest(Object(craft\web\Request))
#10 /srv/users/serverpilot/apps/main-production/web/index.php(24): yii\base\Application->run()
#11 {main}
```

**Comment by kbergha** [comment]
Fixed with #7157 ?

**Comment by brandonkelly** [comment]
Ah yep! Sorry @jalendport, I must have missed your reply.
